plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoSession.java

```diff
@@ -640,6 +646,18 @@ else if (value instanceof Document) {
                 typeSignature = new TypeSignature(StandardTypes.ROW, parameters);
             }
         }
+        else if (value instanceof DBRef) {
+            List<TypeSignatureParameter> parameters = new ArrayList<>();
+
+            TypeSignature idFieldType = guessFieldType(((DBRef) value).getId())
+                    .orElseThrow(() -> new UnsupportedOperationException("Unable to guess $id field type of DBRef from: " + ((DBRef) value).getId()));
+
+            parameters.add(TypeSignatureParameter.namedTypeParameter(new NamedTypeSignature(Optional.of(new RowFieldName(ID)), idFieldType)));
```

Comment:
Let's defined `id` as 3rd field for consistency with DBRef layer model (db -> collection -> id).

Reply:
done

Reply:
done